### ISSUE TITLE
About the implementation of  KLR

### ISSUE THREAD

**Issue by LetheSec** [jaechan-repo/muse_bench#5]
Hi, thanks for your nice work.

I have some questions about the implementation of the `KLR` loss. 
https://github.com/jaechan-repo/muse_bench/blob/6d4fdcbdebe4ad46dccaf70f8526cd23ecff609e/baselines/baselines/iterative.py#L152

If `KLR` refers to KL divergence  as in previous work, when `log_target=True`, it seems that `F.kl_div` should take log probabilities as input instead of using logits directly $^{[1]}$, what impact will this have on the results?

[1] https://pytorch.org/docs/stable/generated/torch.nn.functional.kl_div.html
